docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
-Available threat indicator match data appears in the **Threat Match Detected** subsection.
+Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
+
+Click **Threat intelligence** to expand the **Threat intelligence** sub-tab in the left panel. Within the sub-tab, matched threats are organized into two sections: <<threat-match-detected-section>> and <<enriched-with-threat-intelligence-section>>. Matched threats are ordered in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
 
-[role="screenshot"]
-image::images/matched-indicator-sub-sec.png[Threat Match Detected subsection with matched file hash, 600]
+[discrete]
+[[threat-match-detected-section]]
+==== Threat match detected
+
+The Threat match detected section is solely reserved for threat indicator matches identified by an indicator match rule. Threat indicator matches are produced whenever event data matches a threat indicator field value in your indicator index. If indicator threat matches are not discovered, the section displays a message that none are available.
+
+[discrete]
+[[enriched-with-threat-intelligence-section]]
+==== Enriched with threat intelligence
+
+The Enriched with threat intelligence section shows indicator matches that {elastic-sec} found when querying the alert for fields with threat intelligence. You can use the date time picker to modify the query time frame, which looks at the past 30 days by default. 
+
+Click the **Inspect** button, located on the far right of the threat label, to view more information on the query. If threat matches are not discovered within the selected time frame, the section displays a message that none are available.
```

Comment:
well, I spoke too fast, this section also talk about the no-data message. I'll leave it up to you to keep or remove both.

Reply:
Same comment as above. Though, I'm not sure if the first sentence still applies - @christineweng do you happen to know if the **Inspect** button still exists in the Enriched with threat intelligence section? I can't find an alert that has threat intel enrichments in the test deployment you shared. 

```suggestion
Click the **Inspect** button, located on the far right of the threat label, to view more information on the query.
```

Reply:
@nastasha-solomon inspect button is available, I found an alert with enriched TI data :)
<img width="635" alt="image" src="https://github.com/elastic/security-docs/assets/18648970/e7a17522-f307-4ef8-9447-6b2d3abaae86">

Reply:
Thanks for confirming! I'll keep this line and merge the change.